compiler/luci/lang/src/Nodes/CircleTanh.test.cpp

```diff
@@ -33,22 +33,22 @@ TEST(CircleTanhTest, constructor)
 
 TEST(CircleTanhTest, input_NEG)
 {
-  luci::CircleTanh neg_node;
+  luci::CircleTanh tanh_node;
```

Comment:
Changes in this file has no relation with "U8". It would be better to split this PR.

Reply:
I'll follow your guide. Thank you for your advice! :)